es/refresher-probability.md

```diff
@@ -1,347 +1,347 @@
 **1. Probabilities and Statistics refresher**
 
-&#10230;
+&#10230; Repaso de Probabilidades y Estadísticas
 
 <br>
 
 **2. Introduction to Probability and Combinatorics**
 
-&#10230;
+&#10230; Introducción a las Probabilidades y Combinatoria
 
 <br>
 
 **3. Sample space ― The set of all possible outcomes of an experiment is known as the sample space of the experiment and is denoted by S.**
 
-&#10230;
+&#10230; Espacio Muestral - El conjunto de todos los posibles resultados de un experimento es conocido como el espacio muestral del experimento y se denota como S.
 
 <br>
 
 **4. Event ― Any subset E of the sample space is known as an event. That is, an event is a set consisting of possible outcomes of the experiment. If the outcome of the experiment is contained in E, then we say that E has occurred.**
 
-&#10230;
+&#10230; Evento - Cualquier subconjunto de el espacio muestral es conocido como un evento. Esto significa que un evento es un conjunto de posibles resultados de un experimento. Si el resultado de un experimento esta contenido en E, entonces decimos que el evento E ha ocurrido.
 
 <br>
 
-**5. Axioms of probability For each event E, we denote P(E) as the probability of event E occuring.**
+**5. Axioms of probability - For each event E, we denote P(E) as the probability of event E occuring.**
 
-&#10230;
+&#10230; Axiomas de la probabilidad - Para cada evento E, denotamos P(E) como la probabilidad de que el evento E ocurra.
 
 <br>
 
 **6. Axiom 1 ― Every probability is between 0 and 1 included, i.e:**
 
-&#10230;
+&#10230; Axioma 1 - Cada probabilidad tiene valores entre 0 y 1 inclusive, esto es:
```

Comment:
It must be referring to a closed or inclusive group. Meaning 0 and 1 can be values of probability. Hence, I suggest:

"Cada probabilidad tiene valores inclusivos entre 0 y 1, esto es:"